src/main/java/edu/kis/vh/nursery/CountingOutRhymer.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.vh.nursery;
+
+import edu.kis.vh.nursery.containers.IntContainer;
+import edu.kis.vh.nursery.containers.IntLinkedList;
+
+// kombinacja klawiszy alt + ← oraz → nawiguje pomiedzy otwartymi klasami
+// zmiana wymaga podania implementacji IntContainer
+public class CountingOutRhymer {
+
+  private IntContainer intContainer;
+
+    public CountingOutRhymer(IntContainer intContainer) {
+        super();
+        this.intContainer = intContainer;
+    }
+
+    public CountingOutRhymer() {
```

Comment:
PLEASE FIX: Remove default constructor and fix places that use it (e.g. tests, factories)